lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +222,30 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        sentences = split_by_sentence(text)
+        conllu_sent = []
+        for id, sentence in enumerate(sentences, start=1):
+            conllu_sent.append(ConlluSentence(id, sentence, []))
+        return conllu_sent
+
 
     def run(self) -> None:
         """
         Performs basic preprocessing and writes processed text to files
         """
+        articles = self._corpus.get_articles().values()
+        for article in articles:
+            filled_article = from_raw(self._corpus.path_to_raw_txt_data, article)
```

Comment:
this should be done in corpus manager during initialization. it has a storage for that, right?

Reply:
yes, you are right
I've deleted this strings